roomserver/storage/postgres/storage.go

```diff
@@ -39,7 +41,8 @@ type Database struct {
 func Open(dataSourceName string) (*Database, error) {
 	var d Database
 	var err error
-	if d.db, err = sql.Open("postgres", dataSourceName); err != nil {
+	logrus.WithField("yep", "no").Info("NOT ENTIRELY CRAZY")
```

Comment:
This probably doesn't want to be in the final merge :D

Reply:
Haha, noooo my secret logging lines!